pages/public/contact_us.vue

```diff
@@ -90,12 +101,13 @@ export default {
         name: '',
         email: '',
         message: '',
+        cellphone: '',
       },
       nameRules: [v => !!v || 'Digite seu nome'],
       messageRules: [v => !!v || 'Digite uma mensagem'],
+      cellphoneRules: [v => !!v || 'Digite seu celular com o DDD'],
       emailRules: [
-        v => !!v || 'Digite o e-mail',
-        v => /.+@.+\..+/.test(v) || 'E-mail inválido',
+         v => (v === '' || /.+@.+\..+/.test(v)) || 'E-mail inválido',
```

Comment:
```suggestion
         v => (v === '' || /^[a-zA-Z0-9\._-]+@[a-zA-Z0-9_-]+(\.[a-zA-Z]+)+$/.test(v)) || 'E-mail inválido',
```
Melhoria na expressão regular do e-mail.